RFCs/FS-1112-obsolete-allowed-to-use-obsolete.md

```diff
@@ -0,0 +1,114 @@
+# F# RFC FS-1112 - Obsolete allowed to use Obsolete
+
+<!--The design suggestion [Obsolete allowed to use Obsolete](https://github.com/fsharp/fslang-suggestions/issues/1055) has been marked "approved in principle".
+-->
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/1055)
+- [ ] Approved in principle
+- [ ] [Implementation](https://github.com/dotnet/fsharp/pull/FILL-ME-IN)
+- [ ] Design Review Meeting(s) with @dsyme and others invitees
+- [Discussion](https://github.com/fsharp/fslang-design/discussions/FILL-ME-IN)
+
+# Summary
+
+The general idea is to allow types and functions marked with `System.ObsoleteAttribute` to use other `Obsolete`-marked members without
```

Comment:
```suggestion
The general idea is to allow modules, types and functions marked with `System.ObsoleteAttribute` to use other `Obsolete`-marked members without
```

Reply:
If mentioning Modules separetely, then I guess i should add all combinations related to them to the Design section?